bagua-core-internal/src/comm_ops/decentralized_low_precision_synchronous.rs

```diff
@@ -0,0 +1,231 @@
+use crate::comm_ops::CommOpTrait;
+use crate::communicators::{BaguaCommunicator, BaguaHierarchicalCommunicator, NCCLGroupGuard};
+use crate::datatypes::{BaguaBucket, BaguaTensorRaw, RawBaguaTensor, TensorCompressionMethod};
+use crate::comm_ops::decentralized_full_precision_synchronous::PeerSelectionMode;
+use crate::resource_pool::CUDA_DEVICE_MEMORY_POOL;
+use crate::events::BaguaEventChannel;
+use crate::{BaguaCommOpChannels, BaguaScheduledCommOp};
+use parking_lot::Mutex;
+use std::sync::Arc;
+
+#[derive(Debug)]
+pub struct DecentralizedLowPrecisionSynchronousPostStep {
```

Comment:
"post_step" etc. are planned to be deprecated to simplify operator concept

Reply:
About the `then()` design, is it possible to share states between different operators?